app/components/forms/wizard/basic-details-step.js

```diff
@@ -106,6 +106,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return filter(this.get('data.event.tickets').toArray(), ticket => ticket.get('type') === 'paid').length > 0;
   }),
 
+  hasDonationTickets: computed('data.event.tickets.[]', function() {
```

Comment:
Since this file has not been refactored for ES6 changes, maybe you could do that in this PR too?

Reply:
A good idea, Maybe.

Reply:
@kushthedude Just use a decorator for this computed property. That will suffice.

Reply:
@mrsaicharan1 Try adding the decorator, Computed property stopped working 
```js
@computed('data.event.tickets.[]')
  get hasDonationTickets() {
    return filter(this.get('data.event.tickets').toArray(), ticket => ticket.get('type') === 'donation').length > 0;
  },
```
Anything you see ?